src/routes/post.js

```diff
@@ -1,16 +1,20 @@
 import {subscribe} from '../api/anon-email-lists';
 import {send} from '../api/anon-email-svc';
-import SpoorClient from '@financial-times/n-spoor-client';
+import SpoorClient from '@financial-times/spoor-client';
 import logger from '@financial-times/n-logger';
 
 export default function (req, res, next) {
 
 	const mailingList = req.body && req.body.mailingList ? req.body.mailingList : 'light-signup';
+	const cookies = req.get('cookie') || req.get('ft-cookie-original');
+	const ua = req.get('user-agent');
+
 	const spoor = new SpoorClient({
 		source: 'newsletter-signup',
 		category: 'light-signup',
 		product: req.body && req.body.source ? req.body.source : null,
-		req
+		cookies,
+		ua,
```

Comment:
Our (i.e. the apps team's) Javascript style is tending towards [enforcing trailing commas](https://github.com/Financial-Times/ft-facebook-instant/blob/master/.eslintrc.json#L12) for multiline objects and arrays since it produces cleaner diffs.